litmus-portal/frontend/src/views/Community/CommunityTimeSeriesPlot/styles.ts

```diff
@@ -6,17 +6,25 @@ const useStyles = makeStyles((theme) => ({
     margin: theme.spacing(0.5),
     height: '2.5rem',
     width: '19.375rem',
+    [theme.breakpoints.down('sm')]: {
+      width: '15rem',
+    },
   },
+
   plotCard: {
     paddingTop: theme.spacing(2),
     paddingBottom: theme.spacing(2),
   },
+
   root: {
     height: '2.5rem',
     padding: theme.spacing(0.5),
   },
 
   plot: {
+    margin: 'auto',
+    width: '150%',
+    paddingLeft: '4.5%',
```

Comment:
Please remove all the changes in this file as well, this is what they will result in:

Styles added locally:
![Screenshot from 2021-03-12 18-44-45](https://user-images.githubusercontent.com/40641427/110944868-098a6d80-8363-11eb-8c59-65f137384910.png)

output in 1920x1080 (1080p) screen:
![Screenshot from 2021-03-12 18-43-30](https://user-images.githubusercontent.com/40641427/110944933-1eff9780-8363-11eb-9d2e-2060888cfa42.png)

output for down breakpoint sm:
![Screenshot from 2021-03-12 18-43-45](https://user-images.githubusercontent.com/40641427/110945118-4eae9f80-8363-11eb-99cb-e8444aee4d94.png)

as the map itself is not responsive, adding responsiveness to the form doesn't not help and increases redundant code

Reply:
As mentioned in the above change request, even if I don't test it using dedvtools, here is the output on my normal screen with these styles
![Screenshot from 2021-03-12 18-52-54](https://user-images.githubusercontent.com/40641427/110945820-2ffcd880-8364-11eb-8561-b6460848a21d.png)

Reply:
Why does the community page needs a change?

Reply:
All other plotly graphs are responsive @arkajyotiMukherjee